Source/WebCore/editing/Editor.cpp

```diff
@@ -3670,6 +3670,8 @@ RefPtr<TextPlaceholderElement> Editor::insertTextPlaceholder(const IntSize& size
     if (!placeholder->parentNode())
         return nullptr;
 
+    document->selection().setCaretVisible(false);
```

Comment:
Thinking more about this, would this get overwritten if `FrameSelection::focusedOrActiveStateChanged` gets called in between insertion and removal?

We may end up needing a list of caret visibility suppression "reasons".